modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,694 @@
+    width: 400,
+    plugins: "advcode",
+    advcode_inline: true,
+    toolbar_mode: 'sliding',
+    toolbar: " undo redo | styles | bold italic underline strikethrough | code "
+});
+----
+
+A workaround does exist to resolve this issue: if the More button disappears, re-sizing the editor viewport forces it to re-display. Switching to Full Screen mode and back counts as re-sizing for this workaround’s purposes.
+
+
+=== In Safari 16.1, the ‘Click for more info’ button in Accessibility Checker dialogs is not outlined when made active
+
+*Accessibility Checker* dialogs for accessibility Errors and Warnings include a _Click for more info_ button: a colored square containing a circled question mark.
+
+When this button is navigated to using keyboard navigation, the button is highlighted and a blue outline appears around the highlighted button.
+
+When Safari 16.1 is the host browser for a {productname} instance, the _Click for more info_ button highlights, but the blue outline does not present when this button is navigated to using keyboard navigation.
+
+This is a presentation issue only. The _Click for more info_ button still works as a button in Safari 16.1, responding as expected when it is clicked or pressed, or when the Return key is pressed after navigating to the button using the Tab key.
+
+There is, as of this release, no specific workaround.
+
+NOTE: If the _Click for more info_ button is navigated to using the Tab key, and then the Return key is pressed to activate the button and load the more information page, the blue outline does appear in Safari 16.1 after the Return key is pressed.
+
+
+=== Checkmark over the current color is not displayed when ‘Text color’ or ‘Background color’ is accessed from the {productname} menu
+
+The *Text color* and *Background color* commands set the color of the selected text and the background behind the selected text respectively.
+
+Depending on the {productname} configuration, these commands can be accessed from the *Format* menu or from the {productname} toolbar, or both.
+
+And, when selected text or the background behind selected text is set to one of the colors available from the swatch presented by both commands, a checkbox appears over that color when these commands are selected and the swatch is opened.
+
+As of {productname} 6.3, however, this checkmark does not appear if the *Text color* or *Background color* commands are opened via the *Format* menu. The checkmark does, however, appear when the equivalent toolbar button menus are opened.
+
+This presentation issue does not affect functionality. Selecting a new *Text color* or *Background color*, either via the *Format* menu or the toolbar, changes the selected text color or the selected text’s background color, as expected.
+
+A workaround is to display the {productname} toolbar with the *Text color* and *Background color* options and access these commands from the toolbar.
```

Comment:
Is this saying that the toolbar is displayed from the color options, which are accessed as commands from the toolbar? Because that doesn't make any sense.

```suggestion
A workaround is to specify the *Text color* and *Background color* using the color swatch UI provided by the `forecolor` and `backcolor` toolbar buttons.
```
This might make more sense, saying that the color should be specified using the toolbar buttons.